logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/IsCallRunningUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package com.wire.kalium.logic.feature.call.usecase
+
+import com.wire.kalium.logic.data.call.CallRepository
+import com.wire.kalium.logic.feature.call.CallStatus
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.map
+
+class IsCallRunningUseCase(
```

Comment:
Could we make the contructor `internal` ? It would be nice to avoid creating instances from clients 🙌